README.md

```diff
@@ -87,78 +87,93 @@ Languages:
 Who is using it? [#madewithbgfx](https://twitter.com/search?q=%23madewithbgfx&src=typed_query&f=live)
 -----------------------------------------------------------------------------------------------------
 
-## Airmech
+## AirMech
 
-http://airmech.com/ AirMech is a free-to-play futuristic action real-time
-strategy video game developed and published by Carbon Games.  
-![airmech](https://www.mobygames.com/images/shots/l/830630-airmech-playstation-4-screenshot-blue-bar-on-your-mech-indicates.jpg)
+[AirMech](https://www.carbongames.com/airmech-strike) is a free-to-play
```

Comment:
I intentionally leave URLs visible so that users know what they are clicking on.

Please make all URLs visible.